<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="api-test" tests="1" time="0.000">
  <testsuite name="get merchant" id="f5fb2e1e-b73c-4fdf-b7f6-0efc2e72aa35" timestamp="2024-05-03T10:51:44.099Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiTest">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 6802a33b-6260-4cd7-b604-7f7ce62a3d9b.]]>
        <![CDATA[Collection name: api-test.]]>
        <![CDATA[Request name: get merchant.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>